Framework/DataHandling/src/LoadEventAsWorkspace2D.cpp

```diff
@@ -160,8 +163,10 @@ void LoadEventAsWorkspace2D::exec() {
   const Kernel::NexusHDF5Descriptor descriptor(filename);
 
   // Load the instrument
-  prog->doReport("Loading instrument");
-  LoadEventNexus::loadInstrument<MatrixWorkspace_sptr>(filename, WS, "entry", this, &descriptor);
+  if (getProperty("LoadNexusInstrumentXML")) {
```

Comment:
you don't need to include this `if` because [LoadEventNexus::loadInstrument](https://github.com/mantidproject/mantid/blob/0a3cc423de768430d3716290b7462df62f22e262/Framework/DataHandling/inc/MantidDataHandling/LoadEventNexus.h#L742) takes care of the logic involving parameter `LoadNexusInstrumentXML`

Reply:
Removed `if` block